### ISSUE TITLE
Add a new sentence embedding method: DeCLUTR

### ISSUE THREAD

**Issue by JohnGiorgi** [Separius/awesome-sentence-embedding#22]
Hi there,

Not sure if this list is still being maintained but if so, might I (shamelessly) recommend adding [DeCLUTR: Deep Contrastive Learning for Unsupervised Textual Representations](https://github.com/JohnGiorgi/DeCLUTR). It is an unsupervised method for learning high-quality sentence embeddings that we recently developed. It is similar to [Sentence Transformers](https://github.com/UKPLab/sentence-transformers) in that it pre-trains a transformer-based language model, but because it is unsupervised, you do not need any labels!

- [GitHub](https://github.com/JohnGiorgi/DeCLUTR).
- [Paper](https://arxiv.org/abs/2006.03659).
- Pretrained models: [small](https://huggingface.co/johngiorgi/declutr-small) and [base](https://huggingface.co/johngiorgi/declutr-base).
